.circleci/config.yml

```diff
@@ -1,9 +1,8 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@6
-  slack: circleci/slack@4.4.2
-  codecov: codecov/codecov@3.2.0
+  hmpps: ministryofjustice/hmpps@7
```

Comment:
version 7 is latest and Matt has added some additional debug so we should be able to see what is happening around the app_version issue